xbmc/utils/Mime.cpp

```diff
@@ -525,8 +536,9 @@ std::string CMime::GetMimeType(const std::string &extension)
     ext = extension.substr(posNotPoint);
   transform(ext.begin(), ext.end(), ext.begin(), ::tolower);
```

Comment:
Could we please add `std::` so it is clear which `transform` we want to use? Maybe even change to `std::tolower`?

Reply:
This is just a question. The code was not touched in this PR, so completely up to you.

Reply:
std::tolower requires the stupid lambda addition. i added std::